model/db-object.php

```diff
@@ -132,6 +132,9 @@ public static function zVsech(bool $zCache = false): array
             foreach ($objekty as $objekt) {
                 self::$objektyZVsech[static::class][$objekt->id()] = $objekt;
             }
+            if ((self::$objekty[static::class] ?? null) == null) {
```

Comment:
Díky za návrh opravy.
Tohle máme už fixnuté pomocí `self::$objekty[static::class] ??= [];`

Poznámka: non-strict comparison přes `==` vyhodnotí prázdné pole a null jako stejné hodnoty, takže tvůj kód by už jednou nastavené prázdné pole nastavoval znovu. Nic by to nerozbilo, píšu to jen pro info.
V tom je PHP jiné než třeba JS, kdy v JS je `console.log([] == null) // false`, ale v PHP `var_dump([] == null); // bool(true)`